ci/launch_cluster.sh

```diff
@@ -13,13 +17,22 @@ CHANNEL="$1"
 VARIANT="$2"
 
 if [ "$VARIANT" == "open" ]; then
-  TEMPLATE="https://s3.amazonaws.com/downloads.dcos.io/dcos/${CHANNEL}/cloudformation/multi-master.cloudformation.json"
+  INSTALLER="https://downloads.dcos.io/dcos/${CHANNEL}/dcos_generate_config.sh"
+  EXTRA_DCOS_CONFIG=""
 else
-  TEMPLATE="https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise-aws-advanced/${CHANNEL}/${VARIANT}/cloudformation/ee.multi-master.cloudformation.json"
+  if [ -z "$DCOS_INSTALLER_TEMPLATE" ]; then
+      echo "Expecting DCOS_INSTALLER_TEMPLATE environment variable"
+  fi
+  if [ -z "$DCOS_LICENSE" ]; then
+      echo "Expecting $DCOS_LICENSE environment variable"
+  fi
+  # Replace {} in the template with the channel
+  INSTALLER=${$DCOS_INSTALLER_TEMPLATE/{\}/$CHANNEL}
```

Comment:
How about using sed?

Reply:
I have nothing against sed, however I generally prefer to avoid invoking an external tool for something bash has a built-in. (again, `bash`, not `zsh` or other shells, but we do have a bash hashbang in the beginning 😉 )